### PR TITLE
config.py now uses cache file in . directory.

### PR DESCRIPTION
**Pull request by nasirhm**
With our current Fedora Infrastructure, we're using a file-based cache storage
we can do the same for our development experience.
while memcached being turned off,
It was trying to write to a file in / root directory resulting in an error (which works fine in the deployed Fedora-infra).

This commit modifies the location of the cache file to create it in the current directory.

It would help developers, who are unable to setup memcached
and would like to use file based cache for development.

Related Issues: #148 #140 
Fixes: #140

### CODE REVIEW

**Comment by t0xic0der** [comment]
@nasirhm : I can use some help regarding the expected change here. :sweat_smile:

**Comment by nasirhm** [comment]
> @nasirhm : I can use some help regarding the expected change here.

This change works fine for now and you can use it by turning off the memcached setting :).

It'll be storing the cache in `./cache.json` file in the current directory.

File: files/config.py [hunk 1]
```diff
@@ -48,7 +48,7 @@
 use_memcached = False # Use a memcached store for greater performance
 
 # JSON cache store location
-json_cache_location = "/var/cache/httpd/mote/cache.json"
+json_cache_location = "./cache.json"
```
Comment (t0xic0der) [inline files/config.py:51]: Just as good as promised. :smile: 

image

**Review by t0xic0der — APPROVED** [review]
Nice work, @nasirhm.

**Comment by nasirhm** [comment]
Thank Youu @t0xic0der for the review, :D

**Comment by nasirhm** [comment]
_Discussed in [2020-11-08 sprint meeting](https://youtu.be/dJ995Zl2EH4)_.

---

It is suggested, that it is a good idea to get it merged as it provides a better development experience for new developers.

cc: @ramyaparimi and @HiraTariq-01 please provide a review on this PR.

**Review by HiraTariq-01 — APPROVED** [review]
Good to go! :D

**Comment by ramyaparimi** [comment]
+1 from me.
working as expected!

**Review by ramyaparimi — APPROVED** [review]

**Comment by nasirhm** [comment]
I think, we're good to merge this one, as currently in Fedora Infrastructure, we're using a [separate and modified `config.py`](https://pagure.io/fedora-infra/ansible/blob/master/f/roles/mote/templates/config.py)

Once the CI runs successfully, we're good to merge it. :D

**Comment by t0xic0der** [comment]
I'll make a merge when the CI check completes. :smile:

**Comment by nasirhm** [comment]
Thank you for merging it @t0xic0der

**Pull request merged**